### PR TITLE
Issue #171 Making release artifacts archive bundles

### PR DESCRIPTION
**Pull request by hferentschik**
A low hanging fruit in preparation for the upcoming release. What exactly should be put into the bundle? Right now I add LICENSE, README.md and the actual binary (similar to what OpenShift does with _oc_). Should it be just the binary? Thoughts? 

P.S. Ignore the first two commits, since this pull request is based on the unmerged doc changes.

### CODE REVIEW

**Comment by hferentschik** [comment]
Any comments on this? This is the last thing separating us from a 1.0.0 alpha.

File: docs/developing.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+```shell
+go run hack/e2e.go -v --test --test_args="--ginkgo.focus=\[Conformance\]" --check_version_skew=false --check_node_count=false
+```
+
+To run a specific Conformance Test, you can use the `ginkgo.focus` flag to filter the set using a regular expression.
+The hack/e2e.go wrapper and the e2e.sh wrappers have a little trouble with quoting spaces though, so use the `\s` regular expression character instead.
+For example, to run the test `should update annotations on modification [Conformance]`, use this command:
+
+```shell
+go run hack/e2e.go -v --test --test_args="--ginkgo.focus=should\supdate\sannotations\son\smodification" --check_version_skew=false --check_node_count=false
```
Comment (praveenkumar) [inline docs/developing.md]: Have we tested if that works or not?

File: docs/developing.md [hunk 2]
```diff
@@ -189,44 +185,26 @@ To remove all generated artifacts and installed dependencies, run:
...
-<a name="build-and-release-a-new-iso"></a>
-### Build and Release a New ISO
+* Update the [CHANGELOG.md](../CHANGELOG.md) with the changes since the last release.
+* Bump the version in the [VERSION](../VERSION)
+* Commit
+* Tag the release locally. Something like:
 
-This step isn't always required. Check if there were changes in the deploy directory.
-If you do this, bump the ISO URL to point to the new ISO, and send a PR.
+        $ git tag -a v1.0.0 -m "1.0.0 Release"
```
Comment (praveenkumar) [inline docs/developing.md]: @hferentschik doesn't make release do this steps?

File: docs/developing.md [hunk 3]
```diff
@@ -189,44 +185,26 @@ To remove all generated artifacts and installed dependencies, run:
...
 
-<a name="bump-the-version-in-the-makefile"></a>
-### Bump the version in the Makefile
+* Push the tag:
 
-See [this PR](https://github.com/kubernetes/minikube/pull/165) for an example.
+        $ git push origin v1.0.0
```
Comment (praveenkumar) [inline docs/developing.md]: @hferentschik even gh-release push the created tag as per version info so this step also might not required.

File: docs/developing.md (hunk 2 above)
Comment (hferentschik) [inline docs/developing.md]: he he - touché. Good point. I wrote this before I understood myself what exactly gh-release does. This is in fact obsolete. Let me clean this up.

Comment (hferentschik) [inline docs/developing.md]: fixed

File: docs/developing.md (hunk 3 above)
Comment (hferentschik) [inline docs/developing.md]: fixed

**Comment by praveenkumar** [comment]
LGTM 👍

File: docs/developing.md [hunk 4]
```diff
@@ -189,44 +185,18 @@ To remove all generated artifacts and installed dependencies, run:
...
-Assemble all the meaningful changes since the last release into the CHANGELOG.md file.
-See [this PR](https://github.com/kubernetes/minikube/pull/164) for an example.
+<a name="cutting-the-release"></a>
+### Cutting the release
 
-<a name="build-and-release-a-new-iso"></a>
-### Build and Release a New ISO
+* Update the [CHANGELOG.md](../CHANGELOG.md) with the changes since the last release.
+* Bump the version in the [VERSION](../VERSION)
+* Commit
```
Comment (budhrg) [inline docs/developing.md]: I would say "Commit your changes with appropriate message"

File: docs/developing.md [hunk 5]
```diff
@@ -189,44 +185,18 @@ To remove all generated artifacts and installed dependencies, run:
...
+  defined in your environment as GITHUB_ACCESS_TOKEN
 
-Assemble all the meaningful changes since the last release into the CHANGELOG.md file.
-See [this PR](https://github.com/kubernetes/minikube/pull/164) for an example.
+<a name="cutting-the-release"></a>
+### Cutting the release
 
-<a name="build-and-release-a-new-iso"></a>
-### Build and Release a New ISO
+* Update the [CHANGELOG.md](../CHANGELOG.md) with the changes since the last release.
```
Comment (budhrg) [inline docs/developing.md]: Extra full stop at the end of sentence.

Comment (hferentschik) [inline docs/developing.md]: ?

Comment (budhrg) [inline docs/developing.md]: @hferentschik , the following lines doesn't have full stop. So annoying me due to inconsistencies :smile:

**Comment by budhrg** [comment]
LGTM :+1:  except few comments which might go in next PR if required.

File: docs/developing.md (hunk 5 above)
Comment (hferentschik) [inline docs/developing.md]: ok

**Pull request merged**